livekit-agents/livekit/agents/pipeline/agent_output.py

```diff
@@ -186,49 +185,21 @@ async def _synthesize_task(self, handle: SynthesisHandle) -> None:
             await utils.aio.gracefully_cancel(synth)
 
 
-@utils.log_exceptions(logger=logger)
```

Comment:
I figured it's cleaner to just have one method, and convert the others into async if they're not.